build.sbt

```diff
@@ -58,6 +57,7 @@ lazy val root = (projectMatrix in file("."))
     name := "atum-service-root",
     javacOptions ++= Seq("-source", "1.8", "-target", "1.8", "-Xlint"),
     publish / skip := true,
+    crossScalaVersions := Nil,
     mergeStrategy
```

Comment:
Are we using `sbt assembly` for something?

Reply:
I guess that there will be Jar containing server and its configuration, and the model, nicely packaged together with it

Reply:
I wanna make a bold comment that you do not need `mergeStrategy` at all. Or at least I have not seen it.